test/mxarray.jl

```diff
@@ -124,10 +124,10 @@ a_mx = mxarray(3.25)
 @test jscalar(a_mx) == 3.25
 delete(a_mx)
 
-a_mx = mxarray(@compat Int32(12))
-@test eltype(a_mx) == Int32
+a_mx = mxarray(12)
+@test eltype(a_mx) == Int
 @test size(a_mx) == (1, 1)
-@test jscalar(a_mx) == @compat Int32(12)
+@test jscalar(a_mx) == Int(12)
```

Comment:
Then why not have the test use the native Int? That's most likely what the user will use.